src/main/java/com/deepoove/swagger/diff/compare/OperationDiff.java

```diff
@@ -0,0 +1,167 @@
+package com.deepoove.swagger.diff.compare;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import com.deepoove.swagger.diff.model.ChangedExtensionGroup;
+import com.deepoove.swagger.diff.model.ChangedOperation;
+import com.deepoove.swagger.diff.model.ChangedParameter;
+
+import io.swagger.models.HttpMethod;
+import io.swagger.models.Model;
+import io.swagger.models.Operation;
+import io.swagger.models.Response;
+import io.swagger.models.parameters.Parameter;
+import io.swagger.models.properties.Property;
+
+public class OperationDiff {
+
+  private Map<HttpMethod, Operation> increasedOperations;
+  private Map<HttpMethod, Operation> missingOperations;
+  private Map<HttpMethod, ChangedOperation> changedOperations;
+
+  private Map<HttpMethod, Operation> oldOperations;
+  private Map<HttpMethod, Operation> newOperations;
+  private Map<String, Model> oldDefinitions;
+  private Map<String, Model> newDefinitions;
+  private VendorExtensionDiff extDiffer;
+
+  private boolean hasContractChanges;
+  private boolean hasOnlyCosmeticChanges;
+
+  public OperationDiff(Map<HttpMethod, Operation> oldOperations, Map<HttpMethod, Operation> newOperations,
```

Comment:
Should this be operating on two `Optional<Operation>` instead of the entire map of operations? If not, we should probably call it `OperationsDiff`